components/Button/Button.tsx

```diff
@@ -1,21 +1,24 @@
-export type ButtonProps = {
+import { ButtonType, defaultClassName } from "./utils";
+import clsx from "clsx";
+
+export interface ButtonProps {
   label: string;
-  onClick?: () => void;
+  onClick: () => void;
   className?: string;
-  primary?: boolean;
-};
+  buttonType?: ButtonType;
```

Comment:
sometimes if you have default values for props it is ok to declare default value in the comment. It could increase transparency a liitle for other developers